src/Entity/ClickNCollectShipment.php

```diff
@@ -21,20 +21,14 @@
  */
 trait ClickNCollectShipment
 {
-    /**
-     * @ORM\ManyToOne(targetEntity=\CoopTilleuls\SyliusClickNCollectPlugin\Entity\LocationInterface::class)
-     */
+    #[ORM\ManyToOne(targetEntity:\CoopTilleuls\SyliusClickNCollectPlugin\Entity\LocationInterface::class )]
```

Comment:
```suggestion
    #[ORM\ManyToOne(targetEntity: \CoopTilleuls\SyliusClickNCollectPlugin\Entity\LocationInterface::class )]
```

Also, you can add a `use` statement instead of using the FQCN.
With recent versions of Doctrine, this can even be removed (this will be guessed using the type);